test/Calendar-test.js

```diff
@@ -76,4 +76,28 @@ describe('Calendar', () => {
 
     asserter(calendar).assertToday();
   });
+
+  it('fails when trying to display day of the week following a given format', () => {
+    const formats = ['', null];
+
+    formats.forEach((format) => {
+      const calendar = (
+        <Calendar date={moment()} onSelect={onSelect} dayOfWeekFormat={format} />
+      );
+  
+      asserter(calendar).assertFailingDayOfTheWeek();
```

Comment:
Is this really just `asserter(calendar).assertDayOfTheWeek('dd')`?

If so you could remove the `assertFailingDayOfTheWeek()` function (which doesn't quite feel right anyway)

...and then is the test name more like 
`day of week format defaults to 'dd' when format is invalid`
than
`fails when trying to display day of the week following a given format`

What do you think?

Reply:
It definitely sounds better